rna.clj

```diff
@@ -0,0 +1,13 @@
+(ns exercises.rna)
+
+(def dna->rna {\G \C
+               \C \G
+               \T \A
+               \A \U})
+
+(defn to-rna
+  [dna]
+  (apply str (map (fn [x]
+                    (assert (contains? dna->rna x))
```

Comment:
Given you have the intention of using `contains` to ensure that the caller is given appropriate arguments, it would benefit a lot of clojure's `:pre` check.

If you create one private `(defn- get-dna)` you could use this feature.

http://blog.fogus.me/2009/12/21/clojures-pre-and-post/

It is not common to see `assert` and `:pre`s on the codebase tho. A second alternative to implement would be use the `dna->rna` map as a function!

You can take advantage that clojure hash-maps implement the call protocol, and use it as a function.

```clojure
(def a {:a 1 :b 2})

(=
  (a :a)
  (get a :a)
  (:a a) ;; keywords are also functions
)
```

This would make this function much more straight forward.